environment/env.go

```diff
@@ -66,7 +66,6 @@ const (
 	PostgresDefaultPort = "5432"
 )
 
-// SetupEnv setup the necessary env
```

Comment:
What is the reason to remove those? It will cause lint warning

Reply:
They were extraneous and wrong in few places. I guess I'll fix them if they are needed by linter..